ros2_migration_readme.md

```diff
@@ -0,0 +1,181 @@
+
+**This file describes steps to perform test on the robot localization package for ROS2 (crystal) and other required stuff.**
+
+**Overview:**
+
+	robot_localization is a package of nonlinear state estimation nodes.
+
+**List of changes from ROS1 to ROS2:**
+
+	The changes has been done by following the Migration guide for ROS2( https://index.ros.org/doc/ros2/Migration-Guide/ ).
+
+	1. Migrated all test.cpp into ROS2 style
+	2. Migrated yaml files into ROS2 style
+	3. Migrated .test files into launch.py in ROS2 style
+	4. Migrated CMakeLists.txt and package.xml
+	5. Created Automated scripts in order to run test cases
+	6. Added .travis.yml file in order to check for CI build
+	7. Followed cosmetics rules and updated the files with cppcheck, cpplint and uncrustify rules.
+
+**Pre-requisites:**
+
+	1. System should have installed crystal distro. Check out installation instructions and tutorials https://index.ros.org/doc/ros2/.
+	2. User should checkout diagnostics pkg & build (Refer steps below).
+
+		Steps to checkout diagnostics pkg & build:
+
+			1. Open the new terminal and run below commands.
+			2. mkdir -p diagnostics_ws/src
+			3. cd ~/diagnostics_ws/src
+			4. git clone git@github.com:vaibhavbhadade/diagnostics.git //Note: branch should be remotes/origin/ros2-devel 
```

Comment:
You can roll this into the command itself:

```
git clone git@github.com:vaibhavbhadade/diagnostics.git -b ros2-devel
```

Reply:
Agreed and fixed on https://github.com/cra-ros-pkg/robot_localization/pull/436/commits/ef2e6123bf5149739a013296292266f6073b3320